projects/control-service/cicd/deploy-testing-pipelines-service.sh

```diff
@@ -55,6 +55,9 @@ if [ "$RUN_ENVIRONMENT_SETUP" = 'y' ]; then
                          --docker-email="versatiledatakit@groups.vmware.com" --dry-run=client -o yaml | kubectl apply -f -
 
     kubectl patch serviceaccount default -p '{"imagePullSecrets":[{"name":"'$secret_name'"},{"name":"'$dockerhub_secretname'"}]}'
+    kubectl create namespace cicd-control
+    kubectl create namespace cicd-deployment
+
```

Comment:
may link to the cicd documentation explaining the deployment namespaces

Reply:
What do you mean by this? 
do you expect there to be docs saying which namespaces the integration tests are using? 
I would think the properties file serves as this type of documentation?